Sources/ShopifyCheckoutKit/CheckoutViewController.swift

```diff
@@ -29,6 +29,7 @@ public class CheckoutViewController: UINavigationController {
 		let rootViewController = CheckoutWebViewController(
 			checkoutURL: url, delegate: delegate
 		)
+		rootViewController.notifyPresented()
```

Comment:
How come we're doing this here and not explicitly in the `present()` method?

Reply:
I couldn't see a particularly nice way of doing it in `present()`.  

I think you could get access to the `rootViewController` via something like `viewController.viewControllers.first`. But then you'd need to cast it from a `UIViewController` to a `CheckoutWebViewController` to be able to access the `notifyPresented` method.

Reply:
It'd be something like 
From
```
from.present(CheckoutViewController(checkout: url, delegate: delegate), animated: true)
```

To
```swift
let viewController = CheckoutViewController(checkout: url, delegate: delegate)
from.present(viewController, animated: true)
(viewController.viewControllers.first as? CheckoutWebViewController)?.notifyPresented()
```

Reply:
Ah okay I see the chain now. The `CheckoutViewController` gets instantiated when from.present() is called - so it's essentially the same thing. 

Sounds good to me